app/shared/components/header/header.component.ts

```diff
@@ -57,6 +62,16 @@ const listFacts = [
   ]
 })
 export class Header {
-  private listCharts:Array<any> = listCharts;
-  private listFacts:Array<any> = listFacts;
+  private listCharts: Array<any> = listCharts;
+  private listFacts: Array<any> = listFacts;
+  private urlPath: string;
+
+  constructor(router:Router) {
+
+    router.subscribe((url) => {
+      router.recognize(url).then((instruction) => {
+        this.urlPath = instruction.component.urlPath;
+      });
+    });
```

Comment:
wouldn't be better to compare component instead of path ( in theory it might be changed in future )
eg: 
```typescript
  constructor(router: Router) {
    router.subscribe((url) => {
      router.recognize(url).then((instruction: Instruction) => {
        this.isOnRootView = instruction.component.componentType === Root;
      });
    });
  }
```